rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/30_synonyms_delete.yml

```diff
@@ -0,0 +1,57 @@
+setup:
+  - skip:
+      version: " - 8.8.99"
+      reason: Introduced in 8.9.0
+  - do:
+      synonyms.put:
+        synonyms_set: test-get-synonyms
+        body:
+          synonyms_set:
+            - synonyms: "hello, hi"
+              id: "test-id-1"
+            - synonyms: "bye => goodbye"
+              id: "test-id-2"
+
+---
+"Delete synonyms set":
```

Comment:
Could we please add a GET request here as well to verify that the synonym set no longer exists?

Reply:
That's a good suggestion, doing that!

Reply:
This served to catch a problem as well as I had not specified refresh, which caused the test to fail. Thanks!